### PR TITLE
Fixed property not displaying correctly on the map

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx [hunk 1]
```diff
@@ -25,8 +39,7 @@ const SinglePropertyMarker: React.FC<React.PropsWithChildren<SinglePropertyMarke
 
   const getIcon = () => {
     if (isOwned) {
```
Comment (devinleighsmith) [inline source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx:41]: @FuriousLlama should this if be changed too?

File: source/frontend/src/hooks/repositories/mapLayer/usePimsPropertyLayer.tsx [hunk 2]
```diff
@@ -58,7 +61,7 @@ export const usePimsPropertyLayer = () => {
       // TODO: Enhance useLayerQuery to allow generics to match the Property types
       const forceCasted = featureCollection as FeatureCollection<
         Geometry,
-        PIMS_Property_Location_View
+        PIMS_Property_Boundary_View
```
Comment (devinleighsmith) [inline source/frontend/src/hooks/repositories/mapLayer/usePimsPropertyLayer.tsx:64]: hmm, is this correct?

File: source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx [hunk 3]
```diff
@@ -37,22 +50,30 @@ const SinglePropertyMarker: React.FC<React.PropsWithChildren<SinglePropertyMarke
...
-          pointFeature as PointFeature<PMBC_FullyAttributed_Feature_Properties>
-        ).properties,
+        pimsLocationFeature: pointFeature.properties,
+        pimsBoundaryFeature: null,
+        fullyAttributedFeature: null,
       });
-    } else {
+    } else if (isPimsBoundary(pointFeature)) {
       mapMachine.mapMarkerClick({
         clusterId: clusterId,
```
Comment (devinleighsmith) [inline source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx:63]: do boundaries have cluster ids?

**Review by devinleighsmith — APPROVED** [review]

File: source/frontend/src/hooks/repositories/mapLayer/usePimsPropertyLayer.tsx (hunk 2 above)
Comment (FuriousLlama) [reply source/frontend/src/hooks/repositories/mapLayer/usePimsPropertyLayer.tsx:64]: yes, this was the whole reason for changing it

File: source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx (hunk 3 above)
Comment (FuriousLlama) [reply source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx:63]: they should, all features (should) have a cluster id

File: source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx (hunk 1 above)
Comment (FuriousLlama) [reply source/frontend/src/components/maps/leaflet/Markers/SingleMarker.tsx:41]: This one is fine given the marker will only change if its not owned.

**Pull request merged**